library/alloc/src/fmt.rs

```diff
@@ -84,6 +84,7 @@
 //!
 //! If a named parameter does not appear in the argument list, `format!` will
 //! reference a variable with that name in the current scope.
+//! (Rust 1.58 and later; in [`panic!`], requires Rust 2021.)
```

Comment:
Please remove this line.  We don't include version numbers in the docs for every difference -- we host versioned copies of the docs (like <https://doc.rust-lang.org/1.57.0/std/macro.format.html>) so people using a particular version can see what that version supports.  And we don't need to document `panic!`'s edition behaviour here -- that's fine just in its own documentation.
```suggestion
```